cmake/templates/BoostConfig.cmake.in

```diff
@@ -39,6 +39,18 @@ if(NOT TARGET "Boost::boost")
         INTERFACE_COMPILE_DEFINITIONS "BOOST_ALL_NO_LIB=1"
     )
   endif()
+
+  if(BOOST_CONFIG_LINK_OPENSSL)
+    hunter_add_package(OpenSSL)
```

Comment:
This file used **after** Boost already built. So I guess the fact that OpenSSL is linked to Boost not used on build stage of Boost? If not then user should call `hunter_add_package(OpenSSL)` himself.